source/taggedalgebraic/taggedunion.d

```diff
@@ -75,15 +75,22 @@ align(commonAlignment!(UnionKindTypes!(UnionFieldEnum!U))) struct TaggedUnion
 	}
 
 	static foreach (ti; UniqueTypes!FieldTypes)
-		static if (!isUnitType!(FieldTypes[ti])) {
+		static if(!is(FieldTypes[ti] == void))
```

Comment:
I needed this for supporting tagged enums. Apparently enum members without any types are given type `void`